rcdklibs/R/rcdklibs.R

```diff
@@ -9,7 +9,7 @@ require(rJava, quietly=TRUE)
     }
     jars <- list.files(path=paste(lib,pkg,"cont", sep=.Platform$file.sep),
                        pattern="jar$", full.names=TRUE)
-    .jinit(classpath=c(jars))
+    .jinit(classpath=c(jars),parameters="-Xmx4096m")
 }
```

Comment:
Do you think it's reasonable to set a 4GB heap size by default? Would 2GB be more reasonable?

Reply:
Hei,

I believe this patch should not be there anymore after I re-based the branch, as the rcdklibs is not part of the package anymore? 

@Issue: Yes maybe 4GB is quite much. However, I set this value more "out of my pocket" (got a 32GB laptop here). It is more a hack to overcome memory related java exceptions I got, when I calculated, e.g. the KR-fingerprints, for ~2500 molecules in a loop. I do not know, where does it come from, but when I do not explicitly increase the heap-size it always fails. I will probably put a pull request into `rcdklibs` at some point and we might find out what is the root-cause. 

Best regards,

Eric